packages/jest-editor-support/src/TestReconciler.js

```diff
@@ -108,15 +108,24 @@ module.exports = class TestReconciler {
 
   // Do everything we can to try make a one-liner from the error report
   sanitizeShortErrorMessage(string: string): string {
+    if (string.includes("does not match stored snapshot")) {
+      return "Snapshot has changed"
+    }
+
+    if (string.includes("New snapshot was not written")) {
+      return "New snapshot is ready to write"
+    }
+
```

Comment:
These are some special cases, you can't show anything of value for these two cases in a tweet-sized message, so I've added them as special cases.